packages/node-core/src/db/index.ts

```diff
@@ -2,3 +2,5 @@
 // SPDX-License-Identifier: GPL-3.0
 
 export * from './db.module';
+export * from './migration-service';
+export * from './sync-helper';
```

Comment:
I think if we move sequelizeHelper to this directory then we can remove this export